### PR TITLE
clean up code a small amount.

### PR DESCRIPTION
**Pull request by SamTov**
Just cleaned some code after playing with the vision cones

### CODE REVIEW

**Review by christophlohrmann — APPROVED** [review]

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Black Check
         uses: psf/black@stable
         with:
-          version: "22.3.0"
+          version: "latest"
```
Comment (christophlohrmann) [inline .github/workflows/lint.yaml]: you need to change the precommit config in the same way

**Review by christophlohrmann — CHANGES_REQUESTED** [review]

File: .github/workflows/lint.yaml (hunk 1 above)
Comment (SamTov) [reply .github/workflows/lint.yaml]: It still isn't working but I am on it, thanks for the approve.

**Comment by christophlohrmann** [comment]
can you fix the versions in both files? Then one can be sure that if it passes precommit, it will also pass CI

**Comment by SamTov** [comment]
> can you fix the versions in both files? Then one can be sure that if it passes precommit, it will also pass CI

At the moment both files link to `black@stable` which will be the same version for both. I can fix a numeric version but if we allow both to update with each other they will both always pass and update our code to the latest black standard as they go.

**Review by christophlohrmann — APPROVED** [review]

**Pull request merged**